nurseconnect/forms.py

```diff
@@ -119,18 +119,18 @@ def __init__(self, *args, **kwargs):
         # Security questions fields are created dynamically.
         # This allows any number of security questions to be specified
         for index, question in enumerate(questions):
-            self.fields["question_%s" % index] = forms.CharField(
+            self.fields["question_{}".format(index)] = forms.CharField(
                 label=_(str(question)),
                 widget=forms.TextInput(
                     attrs={
                         "max_length": 150,
                         "class": "Form-input",
-                        "placeholder": "Enter " + str(question).lower(),
-                        "for": "sq" + str(index)
+                        "placeholder": "Enter ".format(str(question).lower()),
```

Comment:
I think you're calling `format()` here without anywhere in the string to insert it. I don't really understand why this doesn't make Python raise an error, I feel like it should.

```
>>> 'string'.format(1)
'string'
>>> 'string {0}'.format(1)
'string 1'
```

Reply:
Good catch! Thanks :)